app/controllers/api/events_controller.rb

```diff
@@ -73,5 +76,17 @@ def confirm_url
     def request_body
       @request_body ||= JSON.parse(request.body.read)
     end
+
+    def verify_request_authenticity
+      head :unauthorized if request_body.blank? || !message_verifier.authenticate!(request_body.to_json)
```

Comment:
Could you add the logic on the right side of the if statement to a named variable please.

```ruby
inauthentic_request = request_body.blank? || !message_verifier.authenticate(request_body.to_json)

head :unauthorised if inauthentic_request
```